server/db/driver/pg/matches.go

```diff
@@ -247,9 +247,62 @@ func userMatches(ctx context.Context, dbe *sql.DB, tableName string, aid account
 	return rowsToMatchData(rows, includeInactive)
 }
 
+func rowsToMatchData(rows *sql.Rows, includeInactive bool) ([]*db.MatchData, error) {
+	defer rows.Close()
+
```

Comment:
This is an unmodified copy of the original `rowsToMatchData`, just moved up here where it is still used.  The new `rowsToMatchDataWithCoins` below shows the modifications.